docs/Documentation/Systems/Swift/running.md

```diff
@@ -28,6 +28,7 @@ The most up to date list of partitions can always be found by running the `sinfo
 |-----------|-------------|
 | long      | jobs up to ten days of walltime |
 | standard  | jobs up to two days of walltime | 
+| GPU       | jobs up to two days of walltime |
```

Comment:
I think the description should also include a brief description of the GPUs available (see Eagle's partitions table for an example)

Reply:
The description about gpu nodes are included.

Reply:
gpu (Nodes with four NVIDIA A100 40 GB Computational Accelerators for GPU-based software)